server/views/components/tweet-block/index.njk

```diff
@@ -0,0 +1,18 @@
+<div class="tweet-block">
+  <div class="row row--cream row--has-wobbly-background {{ {s:10, m:10, l:10} | spacingClasses({padding: ['top', 'bottom']}) }}">
+    <div class="row__wobbly-background"></div>
+    <div class="container">
+      {% componentV2 'divider', null, {'dashed': true} %}
+      <div class="tweet-block__intro flex flex--v-center flex--h-center {{ {s:4, m:4, l:4} | spacingClasses({margin: ['top']}) }}">{% icon 'social/twitter' %}Follow us on Twitter</div>
+      <a href="https://twitter.com/explorewellcome" class="tweet-block__handle flex flex--v-center flex--h-center {{ {s:6, m:6, l:6} | spacingClasses({margin: ['bottom']}) }}">@explorewellcome</a>
+      <div class="grid {{ {s:10, m:10, l:10} | spacingClasses({padding: ['bottom']}) }}">
+        {% for tweet in model.tweets %}
+          <div class="{{ {s:4, m:4, l:6, xl:3} | gridClasses }}">
+            {% componentV2 'tweet', tweet %}
+          </div>
+        {% endfor %}
+      </div>
+      {% componentV2 'divider', null, {'dashed': true} %}
+    </div>
+  </div>
+</div>
```

Comment:
It is a little, yeah. I think it's a necessary evil of composing classes (especially with the filters which stick out like a sore thumb in the html).